pkg/memory-monitor/README.md

```diff
@@ -0,0 +1,300 @@
+`/persist/memory-monitor/output` directory. The output directory is created
+automatically if it does not exist.
+
+The output directory contains:
+* Subdirectory for the last event triggered. The name of the subdirectory is
+  the timestamp of the event. The subdirectory contains:
+  * `event_info.txt`: A text file that contains metadata about the event. It
+    includes the type of the event and the EVE version that was running when the
+    event was triggered.
+  * `allocations_pillar.out`: A list of memory allocation sites of the zedbox
+    process.
+  * `heap_pillar.out`: A heap dump of the zedbox process. It's collected using
+    the built-in go tool. It's a text file that can be either read manually or,
+    even better, analyzed using the `pprof` tool. How to analyze the heap dump
+    is described in the next section.
+  * `memstats_pillar.out`: A memory usage report of the pillar cgroup. It
+    contains the total memory usage of the pillar cgroup according to the cgroup
+    itself, including the cache and the total Resident Set Size (RSS)
+    according to the smaps file of all processes in the cgroup. It also contains
+    the RSS of all processes in the pillar cgroup.
+  * `memstat_eve.out`: A memory usage report of the EVE cgroup. It contains the
+    total memory usage of the EVE cgroup according to the cgroup itself,
+    including the cache and the total Resident Set Size (RSS) according to the
+    smaps file. It also contains the RSS and per-mapping details of all the
+    processes in the EVE cgroup.
+  * `zedbox`: A symlink to the zedbox binary that was used to collect the heap
+    dump.
+* Tar archives of the previous event directories. The archives are created
+  automatically by the handler script when a new event is triggered. The
+  archives are created to save disk space (each event directory can be quite
+  large, ~5-15Mb, while the archive is usually ~1Mb). We keep not more than
+  100 Mb of archives: the oldest archives are deleted by the handler script
+  when the total size of the archives exceeds 100 Mb.
+  The archive does not contain the `zedbox` symlink.
+* `events.log`: A log file that contains a timestamped list of all memory
+  events, archives for which are still present in the output directory.
+* `memory-monitor-handler.log`: A log file that contains the output of the
+  handler script. It  is useful for debugging the handler script if it fails.
+  It's cleared if the handler script run is successful.
+
+## Logs of the memory monitor
+
+The memory monitor logs its output to the syslog. Unfortunately, the syslog
+logs are not shown with the `logread` command. Nevertheless, you can see them
+in the logs files in `/persist/newlogd` directory. The memory monitor logs
+are prefixed with the `memory-monitor` tag.
+
+## How to analyze the heap dump
+
+The heap dump (`heap_pillar.out`) is a text file that can be analyzed using the
+`pprof` tool. The `pprof` tool is a part of the Go toolchain, so it is not
```

Comment:
```suggestion
`go tool pprof` tool. The `go tool pprof` tool is a part of the Go toolchain, so it is not
```
Otherwise people might come up with the idea to use https://github.com/gperftools/gperftools

Reply:
Will fix, thanks

Reply:
Fixed